docs/guides/authentication.md

```diff
@@ -97,9 +97,44 @@ Restart your Keystone App once more, and try to visit <http://localhost:3000/adm
 
 Finally; login with the newly created `User`'s credentials.
 
-## API access control
```

Comment:
I merged this with the section above, but maybe it should be kept for redundancy?